soc/espressif/esp32s2/memory.h

```diff
@@ -7,8 +7,8 @@
 /* SRAM0 (32k) with adjacted SRAM1 (288k)
  * Ibus and Dbus address space
  */
-#define SRAM_IRAM_START  0x40020000
-#define SRAM_DRAM_START  0x3ffb0000
+#define SRAM_IRAM_START  (SRAM_DRAM_START + IRAM_DRAM_OFFSET)
```

Comment:
You must define `SRAM_DRAM_START` first

Reply:
In fact not Lucas, as all those defines will be resolved in a subsequent moment, their definition order don't matter.